modules/osd-create-cluster-ccs.adoc

```diff
@@ -284,6 +306,13 @@ In the event of critical security concerns that significantly impact the securit
 ====
 
 . Review the summary of your selections and click *Create cluster* to start the cluster installation. The installation takes approximately 30-40 minutes to complete.
++
+ifdef::osd-on-gcp[]
+[NOTE]
+====
+If you delete a cluster that was installed into a GCP shared VPC, you must inform the VPC owner of the host project to remove the service account and the roles granted to the service account that was referenced during the creation of the cluster.
```

Comment:
In fact, the service account itself would be deleted during the cluster deletion. So suggest to put it like below, 

... you must inform the VPC owner of the host project to remove the IAM policy binding of the service account and its binding roles in the host project.

Reply:
Hello @svmrh,
For point #1, I have added a note to that doc: 
![image](https://github.com/openshift/openshift-docs/assets/122639474/e7c57191-4fb6-45db-965d-a73739f86747)

Reply:
For point #2, I have added a note: 
![image](https://github.com/openshift/openshift-docs/assets/122639474/97ce29c4-cf71-4bd2-b66c-edb02320bb55)

Reply:
> In fact, the service account itself would be deleted during the cluster deletion. So suggest to put it like below,
> 
> ... you must inform the VPC owner of the host project to remove the IAM policy binding of the service account and its binding roles in the host project.

Thanks @jianli-wei,
   I applied your suggestion with just a slight tweak in wording to fit the narrative prior to that note. Thanks for the clarity.   
![image](https://github.com/openshift/openshift-docs/assets/122639474/bb4a7c3f-64f5-46f3-a5b5-90eb2d38c536)